src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/PackageReadmeControl.xaml.cs

```diff
@@ -75,19 +109,22 @@ private void UserControl_DataContextChanged(object sender, DependencyPropertyCha
 
         private void PackageReadmeControl_Unloaded(object sender, RoutedEventArgs e)
         {
-            if (ReadmeViewModel is not null)
+            if (ReadmeViewModel is not null && _markdownPreview is not null)
             {
                 ThreadHelper.JoinableTaskFactory.Run(async () =>
                 {
                     await _markdownPreview.UpdateContentAsync("", ScrollHint.None);
                 });
-                ReadmeViewModel.PropertyChanged -= ReadmeViewModel_PropertyChanged;
             }
         }
 
         private void PackageReadmeControl_Loaded(object sender, RoutedEventArgs e)
         {
-            NuGetUIThreadHelper.JoinableTaskFactory.Run(UpdateMarkdownAsync);
+            _ = NuGetUIThreadHelper.JoinableTaskFactory.RunAsync(async () =>
+            {
+                await InitializeAsync();
```

Comment:
You need a cancellation token representing the lifetime of this JoinableTaskFactory:

[❌ DO NOT call JoinableTaskFactory.RunAsync without passing a cancellation token based on its owner's lifetime](
https://devdiv.visualstudio.com/DevDiv/_wiki/wikis/DevDiv.wiki/24027/Threading?anchor=%E2%9D%8C-**do-not**-call-%60joinabletaskfactory.runasync%60-without-passing-a-cancellation-token-based-on-its-owner%27s-lifetime)

Reply:
Turns out this is the default factory, so this rule doesn't apply but do cancellation token comment around InitializeAsync.